pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "rocketpy"
+version = "1.2.2"
+description="Advanced 6-DOF trajectory simulation for High-Power Rocketry."
+dynamic = ["dependencies", "optional-dependencies"]
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+  {name = "Giovani Hidalgo Ceotto", email = "ghceotto@gmail.com"}
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Topic :: Rocket Flight Simulation :: Libraries :: Python Modules",
+]
+
+[project.urls]
+homepage = "https://rocketpy.org/"
+documentation = "https://docs.rocketpy.org/"
+repository = "https://github.com/RocketPy-Team/RocketPy"
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+py-modules = []
+
+[tool.setuptools.package-data]
+drag_curves = ["**/*.csv"]
+engine_files = ["**/*.eng"]
+environment_files = ["**/*.nc"]
+jupyter_notebooks = ["**/*.ipynb"]
+documentation = ["**/*.rst"]
```

Comment:
What is the expected behavior for these lines?

I understand this will install data files every time someone installs rocketpy package (e.g. `pip install rocketpy`). Is that right?

Should that be a desired behavior? I mean, we have been avoiding to install unnecessary dependencies along with rocketpy since we want to keep the download time as short as possible, requiring all these csv, nc, ipynb, rst files to be downloaded seems a bit too much imo. Of course we can discuss that.
 
```
[tool.setuptools.package-data]
drag_curves = ["**/*.csv"]
engine_files = ["**/*.eng"]
environment_files = ["**/*.nc"]
jupyter_notebooks = ["**/*.ipynb"]
documentation = ["**/*.rst"]
```

Reply:
That is my reading of the package-data tag as well. 

But I agree we shouldn't pack useless data. I just assumed these files were somehow dependencies of the rocketpy lib as they share space in the same repo. 

Since they are not I will delete these lines, thanks for bringing this up!